packaging/linux/post_install.sh

```diff
@@ -173,6 +173,14 @@ fix_bad_config_perms() {
 
 fix_bad_config_perms
 
+# Associate Keybase with .saltpack files.
+if command -v xdg-mime &> /dev/null ; then
+  # Associate .saltpack with application/x-saltpack.
+  xdg-mime install --mode system /opt/keybase/x-saltpack.xml
```

Comment:
`xdg-mime install` is hardcoded to copy the file into the first directory in $XDG_DATA_DIRS (defaulting to /usr/local/share/:/usr/share/) which contains a mime/packages/ directory, then run update-mime-database.

When building distribution packages you MUST not use this. Instead, simply install it directly to `/usr/share/mime/packages/` so that it will be uninstalled when removing the package with rpm/apt/pacman.

Optionally, run `update-mime-database` the same way you run `gtk-update-icon-cache`, for package managers which do not provide automatic handling of this (pacman handles both automatically).